src/servicebroker-npm/.vscode/settings.json

```diff
@@ -17,6 +17,9 @@
   "[javascript]": {
     "editor.formatOnSave": false
   },
+  "[typescript]": {
+    "editor.formatOnSave": false
```

Comment:
This interferes with line 24 - eslint fixes, so I disabled it.

Reply:
Without this, a great many style errors won't fix themselves, so in the rare case of the save formatting being in conflict with the eslint rules, we find another syntax that doesn't fight.
FWIW though, I checked out your PR, reverted this change, and then re-saved your 3 changed/added .ts files, and nothing was reformatted, so I don't see the problem that this change was intending to fix.

Reply:
Weird, for me there were very significant collisions all the time. The red squigles would never go away, no matter how many times I saved it.

I think the style errors get fixed by eslint [through prettier](https://github.com/microsoft/vs-servicehub/blob/e4dd935aebd6097dea112ba69ea74fb7876e6211/src/servicebroker-npm/.eslintrc.json#LL26-L37C4), instead of the editor fixing them.

I think what's happening is that I have a different prettier config in my user settings for VSCode, which the workspace settings don't override - which is why I'm seeing conflicts because eslint is also trying to format these using a different set of prettier rules provided in the eslintrc.json config.

Reply:
If you can send a PR that makes more workspace settings explicit so that it works given a more diverse set of user-level settings, I'd be happy to take that.